x-pack/plugins/observability_onboarding/public/assets/standalone_agent_setup.sh

```diff
@@ -67,7 +68,7 @@ updateStepProgress() {
 ELASTIC_AGENT_DOWNLOAD_URL="https://artifacts.elastic.co/downloads/beats/elastic-agent/${artifact}.tar.gz"
 echo "Downloading Elastic Agent archive from ${ELASTIC_AGENT_DOWNLOAD_URL}"
 updateStepProgress "ea-download" "loading"
-curl -L -O $ELASTIC_AGENT_DOWNLOAD_URL --fail
+echo "curl -L -O $ELASTIC_AGENT_DOWNLOAD_URL";sleep 1 # curl -L -O $ELASTIC_AGENT_DOWNLOAD_URL --fail
```

Comment:
don't forget to revert this locally if testing, i do this so i don't have to re-download the agent each time :D